Where2Play/Models/MusicBrainz.cs

```diff
@@ -6,110 +6,109 @@
+    public LifeSpan LifeSpan { get; set; }
 
-        [JsonProperty("sort-name")]
-        public string SortName { get; set; }
+    [JsonProperty("area")]
+    public Area Area { get; set; }
 
-        [JsonProperty("end-area")]
-        public object EndArea { get; set; }
+    [JsonProperty("sort-name")]
+    public string SortName { get; set; }
 
-        [JsonProperty("country")]
-        public string Country { get; set; }
+    [JsonProperty("end-area")]
+    public object EndArea { get; set; }
 
-        [JsonProperty("gender")]
-        public object Gender { get; set; }
+    [JsonProperty("country")]
+    public string Country { get; set; }
 
-        [JsonProperty("isnis")]
-        public string[] Isnis { get; set; }
+    [JsonProperty("gender")]
+    public object Gender { get; set; }
 
-        [JsonProperty("id")]
-        public Guid Id { get; set; }
+    [JsonProperty("isnis")]
+    public string[] Isnis { get; set; }
 
-        [JsonProperty("type-id")]
-        public Guid TypeId { get; set; }
+    [JsonProperty("id")]
+    public Guid Id { get; set; }
 
-        [JsonProperty("disambiguation")]
-        public string Disambiguation { get; set; }
+    [JsonProperty("type-id")]
+    public Guid TypeId { get; set; }
 
-        [JsonProperty("rating")]
-        public Rating Rating { get; set; }
-    }
+    [JsonProperty("disambiguation")]
+    public string Disambiguation { get; set; }
 
-    public partial class Area
-    {
-        [JsonProperty("id")]
-        public Guid Id { get; set; }
+    [JsonProperty("rating")]
+    public Rating Rating { get; set; }
+}
 
-        [JsonProperty("type-id")]
-        public object TypeId { get; set; }
+public partial class Area
+{
+    [JsonProperty("id")]
+    public Guid Id { get; set; }
 
-        [JsonProperty("disambiguation")]
-        public string Disambiguation { get; set; }
+    [JsonProperty("type-id")]
+    public object TypeId { get; set; }
 
-        [JsonProperty("sort-name")]
-        public string SortName { get; set; }
+    [JsonProperty("disambiguation")]
+    public string Disambiguation { get; set; }
 
-        [JsonProperty("type")]
-        public object Type { get; set; }
+    [JsonProperty("sort-name")]
+    public string SortName { get; set; }
 
-        [JsonProperty("name")]
-        public string Name { get; set; }
-    }
+    [JsonProperty("type")]
+    public object Type { get; set; }
 
-    public partial class LifeSpan
-    {
-        [JsonProperty("ended")]
-        public bool Ended { get; set; }
+    [JsonProperty("name")]
+    public string Name { get; set; }
+}
 
-        [JsonProperty("begin")]
-        [JsonConverter(typeof(ParseStringConverter))]
-        public object Begin { get; set; }
+public partial class LifeSpan
+{
+    [JsonProperty("ended")]
+    public bool Ended { get; set; }
 
-        [JsonProperty("end")]
-        public object End { get; set; }
-    }
+    [JsonProperty("begin")]
+    [JsonConverter(typeof(ParseStringConverter))]
```

Comment:
This looks useful, but why/how?  Add a comment here, in the Files Changed section, to describe the purpose of this JsonConverter.